domain/consensus/processes/ghostdagmanager/ghostdag_test.go

```diff
@@ -110,30 +107,30 @@ func TestGHOSTDAG(t *testing.T) {
 						factory.implName, info.Name(), testBlockData.ID, err)
 				}
 
-				if testBlockData.Score != (ghostdagData.BlueScore) {
+				if testBlockData.Score != (ghostdagData.BlueScore()) {
 					t.Fatalf("\nTEST FAILED:\n Impl: %s, FileName: %s \nBlock: %s, \nError: expected blue score %d but got %d.",
-						factory.implName, info.Name(), testBlockData.ID, testBlockData.Score, ghostdagData.BlueScore)
+						factory.implName, info.Name(), testBlockData.ID, testBlockData.Score, ghostdagData.BlueScore())
 				}
 
-				if *StringToByte(testBlockData.SelectedParent) != *ghostdagData.SelectedParent {
+				if *StringToByte(testBlockData.SelectedParent) != *ghostdagData.SelectedParent() {
 					t.Fatalf("\nTEST FAILED:\n Impl: %s, FileName: %s \nBlock: %s, \nError: expected selected parent %v but got %v.",
-						factory.implName, info.Name(), testBlockData.ID, testBlockData.SelectedParent, string(ghostdagData.SelectedParent[:]))
+						factory.implName, info.Name(), testBlockData.ID, testBlockData.SelectedParent, string(ghostdagData.SelectedParent()[:]))
```

Comment:
hash has a stringer, so you don't need to convert it to string (and this way it's also not reversed)